### PR TITLE
update settings package on master

### PR DESCRIPTION
**Pull request by mandric**
https://github.com/medic/medic-projects/issues/2473

### CODE REVIEW

**Comment by alxndrsn** [comment]
Code looks fine.  Please note that build is broken.

**Comment by mandric** [comment]
Right the build was broken but that was against an old build.  Is it worth rebasing and force push to trigger another test build?

File: packages/kanso-app-settings/build/compile.js [hunk 1]
```diff
@@ -13,6 +13,7 @@ module.exports = {
                 callback(null, doc);
             });
         } else {
+            doc.app_settings = settings;
```
Comment (garethbowen) [inline packages/kanso-app-settings/build/compile.js]: I think we should revert this now.

Comment (mandric) [inline packages/kanso-app-settings/build/compile.js]: Ok, repushed.

**Pull request merged**